### PR TITLE
test cases for anganwadi routes and child routes

### PR DESCRIPTION
**Pull request by AlekhyaArasavalli**
Added Test cases for anganwadi and child model validation and routes 
[#145](https://github.com/thinkarlabs/P23-003-MalnutritionDashboard/issues/145) 
[#142] (https://github.com/thinkarlabs/P23-003-MalnutritionDashboard/issues/142)

### CODE REVIEW

File: Backend/Test_cases/test_childroutes.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+def test_delete_child_invalid_id():
+    # Provide an invalid child ID
+    invalid_child_id = "invalid_id_here"
+    # Send a DELETE request with an invalid child ID
+    response = client.delete(f"/api/deletechild?id={invalid_child_id}")
+    # Assert the response
+    assert response.status_code == 404
+
+
+
```
Comment (mjayakumarmca) [inline Backend/Test_cases/test_childroutes.py:135]: can u remove all these extra lines

**Review by mjayakumarmca — CHANGES_REQUESTED** [review]
just 1 comment

File: Backend/Test_cases/test_childroutes.py (hunk 1 above)
Comment (AlekhyaArasavalli) [reply Backend/Test_cases/test_childroutes.py]: removed.. pls have a look and merge..Thanks!

**Review by mjayakumarmca — APPROVED** [review]
LGTM

**Pull request merged**